test/fixtures/manifests/basic.pp

```diff
@@ -11,6 +11,11 @@
 #                    consisting of 3 lines
 # $typed::           something having it's type explicitly set
 # $multivalue::      list of users
+# $complex_variant:: A Variant type that can be:
+#                    String with:
+#                    '' or 'unmanaged' - Host auth control done elsewhere
+#                    'ip <List of IPs>' - Allowed IPs/ranges
+#                    Array of strings with ip or host as above
```

Comment:
We probably can cut this down to a more artificial doc string.
Currently it very much mimics what's in https://github.com/puppetlabs/puppetlabs-apache/blob/main/manifests/mod/status.pp (and breaking our doc builds)